robyn/scaffold/postgres/requirements.txt

```diff
@@ -0,0 +1,2 @@
+robyn
+psycopg2
```

Comment:
As `pip3 install psycopg2` did not work due to this [reason](https://stackoverflow.com/questions/5420789/how-to-install-psycopg2-with-pip-on-python), so we should change it to `psycopg2-binary` from `psycopg2`


Error when you only run `pip3 install psycopg2`
```
error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [23 lines of output]
      running egg_info
      creating /private/var/folders/p8/gp_5g0jx34vc15g471jmpnpw0000gn/T/pip-pip-egg-info-qzetowdt/psycopg2.egg-info
      writing /private/var/folders/p8/gp_5g0jx34vc15g471jmpnpw0000gn/T/pip-pip-egg-info-qzetowdt/psycopg2.egg-info/PKG-INFO
      writing dependency_links to /private/var/folders/p8/gp_5g0jx34vc15g471jmpnpw0000gn/T/pip-pip-egg-info-qzetowdt/psycopg2.egg-info/dependency_links.txt
      writing top-level names to /private/var/folders/p8/gp_5g0jx34vc15g471jmpnpw0000gn/T/pip-pip-egg-info-qzetowdt/psycopg2.egg-info/top_level.txt
      writing manifest file '/private/var/folders/p8/gp_5g0jx34vc15g471jmpnpw0000gn/T/pip-pip-egg-info-qzetowdt/psycopg2.egg-info/SOURCES.txt'

      Error: pg_config executable not found.

      pg_config is required to build psycopg2 from source.  Please add the directory
      containing pg_config to the $PATH or specify the full executable path with the
      option:

          python setup.py build_ext --pg-config /path/to/pg_config build ...

      or with the pg_config option in 'setup.cfg'.

      If you prefer to avoid building psycopg2 from source, please install the PyPI
      'psycopg2-binary' package instead.

      For further information please check the 'doc/src/install.rst' file (also at
      <https://www.psycopg.org/docs/install.html>).

      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

```